<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>防疫居家隔离监控系统</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1001</width>
     <height>681</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>连接到云端</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>991</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/image/image/medical.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>991</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;云端医疗防疫居家隔离监控系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>380</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>连接到云端</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>环境信息监测</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>961</width>
       <height>621</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>查看统计图：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="weatherStat">
          <property name="text">
           <string>温度统计图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="weatherStat2">
          <property name="text">
           <string> 湿度统计图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="weatherStat3">
          <property name="text">
           <string> 大气压强统计图</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>详细数据：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="weatherTable"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>来访信息监测</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>961</width>
       <height>621</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>查看统计图：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="visitStat">
          <property name="text">
           <string>授权和非法访问次数统计图</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>详细数据：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="visitTable"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>远程通信</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>931</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>远程警告提醒控制</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>851</width>
        <height>93</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="alarmOn">
         <property name="text">
          <string>开启对方远程警报</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="alarmOff">
         <property name="text">
          <string>关闭对方远程警报</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="alarmSend">
         <property name="text">
          <string>确认发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>931</width>
       <height>441</height>
      </rect>
     </property>
     <property name="title">
      <string>远程聊天通信</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>41</x>
        <y>41</y>
        <width>851</width>
        <height>362</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>对方消息：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="piMsg">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>暂无</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="piGet">
         <property name="text">
          <string>拉取消息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>发送消息：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="pcMsg">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pcSend">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string>确认发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
